.travis.yml

```diff
@@ -20,122 +19,71 @@ git:
-        - export HW=cm3
-        - sed -i 's/^java_opt=.*/java_opt=Zulu8-32/' build-image/openhabian.conf
-        - docker build -t openhabian/${HW}-openhabian -f Dockerfile.openhabian-arm32hf-CM3 .
-        - echo -e "\n\e[36mopenHABian test installation 32bit Azul Embedded Java on virtual $HW ...\n"
-        - docker run --memory="1g" --memory-reservation="1g" --name "openhabian-${HW}" -d openhabian/${HW}-openhabian;
         - docker exec -i "openhabian-${HW}" bash -c './build.bash local-test && mv ~/.profile ~/.bash_profile && /etc/rc.local'
-    - env: arm32hf
+    - name: openHABian installation test with 32bit Azul Java 8 on virtual RPi2
+      env: HW=rpi2
       arch: arm64
       script:
-        # 64bit Azul Java on 32bit ARMv8 (RPi3, CM3) on 32bit OS on Travis Arm64 HW
-        # This is known to not work - openHABian is supposed to downgrade to 32bit on install.
-        - export HW=rpi3
-        - sed -i 's/^java_opt=.*/java_opt=Zulu8-64/' build-image/openhabian.conf
-        - docker build -t openhabian/${HW}-openhabian -f Dockerfile.openhabian-arm32hf-native .
-        - echo -e "\n\e[36mopenHABian test installation 64bit Azul Embedded Java on virtual $HW ..."
-        - echo -e "\e[31m ** This is known to not work. openHABian is supposed to force install down to 32bit **\n"
-        - docker run --memory="1g" --memory-reservation="1g" --name "openhabian-${HW}" -d openhabian/${HW}-openhabian;
+        # 32bit Azul Java 8 on a 32bit ARMv7l (RPi2) virtual platform, running in a 32bit OS on Travis Arm64 HW
+        - sed -i 's/^java_opt=.*$/java_opt=Zulu8-32/' build-image/openhabian.conf
+        - docker build -t openhabian/${HW}-openhabian -f Dockerfile.raspberry-pi2 .
+        - echo -e "\n\e[36mopenHABian test installation using 32bit Azul Embedded Java 8 running on a\e[31m 32bit OS\e[36m running in a virtual ${HW}...\e[0m\n"
+        - docker run --memory="1g" --memory-reservation="1g" --name "openhabian-${HW}" -d openhabian/${HW}-openhabian
         - docker exec -i "openhabian-${HW}" bash -c './build.bash local-test && mv ~/.profile ~/.bash_profile && /etc/rc.local'
-    - env: aarch64
+    - name: openHABian installation test with 32bit Azul Java 8 on virtual RPi3
+      env: HW=rpi3
       arch: arm64
       script:
-        # 64bit Azul Java on 64bit ARMv8 (RPi3,4) on 32bit OS on Travis Arm64 HW
-        # This is known to not work - openHABian is supposed to downgrade to 32bit on install.
-        - export HW=rpi3
-        - sed -i 's/^java_opt=.*/java_opt=Zulu8-64/' build-image/openhabian.conf
-        - docker build -t openhabian/${HW}-openhabian -f Dockerfile.openhabian-aarch64-native .
-        - echo -e "\n\e[36mopenHABian test installation 64bit Azul Embedded Java on virtual $HW ..."
-        - echo -e "\e[31m ** This is known to not work. openHABian is supposed to force install down to 32bit **\n"
+        # 32bit Azul Java 8 on a 64bit ARMv8 (RPi3) virtual platform, running in a 32bit OS on Travis Arm64 HW
```

Comment:
I don't think we should do that until we make Java 11 default.

Reply:
No we need building/testing asap, you already do with Adopt, and Java 11 is not far away, that's closer than any 64bit implementation is.
Just read about the first OH3 package being in the works to get built.

Reply:
So a "32bit Azul Java 11 on a 64bit ARMv8 (RPi3) virtual platform, running in a 32bit OS" test instead of "64bit Azul Java 8 on a 64bit ARMv8 (RPi3) virtual platform, running in a 64bit OS"

I can live with that, seeing as we don't recommend 64bit Java on RPi anyway

Reply:
We are trying to slim down on the testing runs, compromises need to be made.

Reply:
yes but please put that 64bit OS test case back in and change the third case instead to be Zulu 11 instead of 8.
There's no relevant diff in case 2 vs 3 (the OS uses the same function set no matter if you install on RPi 2 or 3).
I am thinking to also release a 64bit image (of course clearly tagged to be experimental) but at least it should be tested. Not sure which Java, probably Zulu 11 ?